appinfo/app.php

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
+use OC\User\LoginException;
```

Comment:
I think there should be a fitting Exception or interface in the OCP space, no?

Reply:
[The exception thrown is `OC\User\LoginException`](https://github.com/nextcloud/server/blob/12d39e818d25a4faa632cfd5945f7ef6ec8e8d27/lib/private/legacy/OC_User.php#L160), which [directly inherits from `\Exception`](https://github.com/nextcloud/server/blob/dae7c159f728a90ffa53247d6e033abdae5d2bd6/lib/private/User/LoginException.php#L10), and [is not wrapped and converted to any other exception in `handleApacheAuth()`](https://github.com/nextcloud/server/blob/12d39e818d25a4faa632cfd5945f7ef6ec8e8d27/lib/private/legacy/OC_User.php#L228-L241) :shrug:

Reply:
sheesh :fearful: